src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1343,6 +1344,8 @@ public void deleteNode(UUID studyUuid, UUID nodeId, boolean deleteChildren) {
         startTime.set(System.nanoTime());
         DeleteNodeInfos deleteNodeInfos = new DeleteNodeInfos();
         deleteNodeInfos.setNetworkUuid(networkStoreService.doGetNetworkUuid(studyUuid));
+        boolean invalidateChildrenBuild = !EMPTY_ARRAY.equals(networkModificationTreeService.getNetworkModifications(studyUuid, nodeId));
```

Comment:
you didn't add it, but it's weird getNetworkModifications takes a studyUuid parameter that it never uses